book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
- [ ] You did well here, but we also need a date and a person (what you have done in the method below) to record to know who is renting a book.
```Ruby
def add_rental(person, date)
     Rental.new(date, self, person)
 end
```